README.md

```diff
@@ -42,7 +42,7 @@ nats.request('help', null, {'max':1}, function(response) {
 
 
 // Request for single response with timeout.
-nats.requestOne('help', null, 1000, function(response) {
+nats.requestOne('help', null, {'max':1}, 1000, function(response) {
```

Comment:
The requestOne doesn't require the max option, but accepts any option that can be passed to the underlying subscription. So the correct incantation for the sample should be: 
nats.requestOne('help', null, {}, 1000, function(response) {} or 
nats.requestOne('help, null, null, function(response){}
```